admin-default.properties

```diff
@@ -201,6 +201,8 @@ mosip.iam.module.redirecturi=${mosip.api.internal.url}/v1/admin/login-redirect/
 #mosip.iam.module.login_flow.scope=cls
 mosip.iam.module.login_flow.scope=email
 mosip.iam.module.login_flow.response_type=code
+
+#This is the endpoint use by ui(browser) based applications to redirect to open-id system. This URL should be reachable through the browser.
```

Comment:
Grammar update- This is the endpoint used by UI (browser) based applications to redirect to an openID system. This URL should be reachable through a browser.